python/create_account_with_captcha.py

```diff
@@ -75,13 +90,42 @@ def get_captcha_fields():
             return k and k['fields']
     return None
 
+
+def get_registration_fields():
+    """ Fetch the registration form fields from `authmanagerinfo` module """
+
+    response = S.get(
+        url=MEDIA_API_ENDPOINT,
+        params={
+            'action': 'query',
+            'meta': 'authmanagerinfo',
+            'amirequestsfor': 'create',
+            'format': 'json'})
+
+    data = response.json()
+    query = data and data['query']
+    authmanagerinfo = query and query['authmanagerinfo']
+    fields = authmanagerinfo and authmanagerinfo['requests']
+
+    ret_field = dict()
+
+    for k in fields:
```

Comment:
Variable names in this for loop could be slightly improved and made more self-explanatory. We don't want them to be lengthy either..